### PR TITLE
Update modal styling to match PF4 modal-box

### PR DESCRIPTION
**Pull request by sg00dwin**
https://issues.redhat.com/browse/CONSOLE-1966

Screen Shot 2020-01-06 at 7 02 47 PM

Screen Shot 2020-01-06 at 7 01 29 PM

modal-resize

### CODE REVIEW

File: frontend/public/components/factory/modal.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const ModalTitle: React.SFC<ModalTitleProps> = ({
   className = 'modal-header',
 }) => (
   <div className={className}>
-    <h4 className="modal-title">{children}</h4>
+    <h1 className="pf-c-title pf-m-2xl">{children}</h1>
```
Comment (rhamilto) [inline frontend/public/components/factory/modal.tsx]: This change is going to break:
* https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts#L5
* https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/integration-tests/views/editCDView.ts#L14

Comment (sg00dwin) [reply frontend/public/components/factory/modal.tsx]: updated tests

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { $ } from 'protractor';
 
 export const flavorDropdownId = '#vm-flavor-modal-flavor-dropdown';
 
-export const modalTitle = () => $('.modal-title');
+export const modalTitle = () => $('.pf-c-title');
```
Comment (rhamilto) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts]: This is a bit generic.  Perhaps it'd be better to add something like `data-test-id="modal-title"` to `modal.tsx`.

Comment (sg00dwin) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts]: Yeah, do like that better

File: frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { $ } from 'protractor';
 
 export const flavorDropdownId = '#vm-flavor-modal-flavor-dropdown';
 
-export const modalTitle = () => $('.modal-title');
+export const modalTitle = () => $('#modal-title');
```
Comment (rhamilto) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts]:
```suggestion
export const modalTitle = () => $('[data-test-id="modal-title"]');
```

File: frontend/packages/kubevirt-plugin/integration-tests/views/editCDView.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@ export const cdTypeSelect = (number) => $(`#cd-rom-modal-select-type-cd-drive-${
 export const cdStorageClassSelect = (number) => $(`#cd-url-storageclass-input-cd-drive-${number}`);
 export const cdPVCSelect = (number) => $(`#cdrom-pvc-input-cd-drive-${number}`);
 
-export const modalTitle = $('.modal-title');
+export const modalTitle = $('#modal-title');
```
Comment (rhamilto) [inline frontend/packages/kubevirt-plugin/integration-tests/views/editCDView.ts]: export const modalTitle = () => $('[data-test-id="modal-title"]');

**Comment by spadgett** [comment]
/hold
until #3747 merges

**Comment by spadgett** [comment]
/hold
to avoid retest while CI is broken

**Comment by sg00dwin** [comment]
retest

**Comment by spadgett** [comment]
/hold
for merge queue fix #4065

**Pull request merged**